armi/reactor/converters/uniformMesh.py

```diff
@@ -588,17 +592,16 @@ def _computeAverageAxialMesh(self):
         The reason for this is to account for the fact that multiple assemblies (i.e., fuel assemblies)
         may have a different mesh due to differences in thermal and/or burn-up expansion.
         """
+
         src = self._sourceReactor
         refAssem = src.core.refAssem
-        # Get the number of reference mesh points, including the sub-mesh. This subtracts 1
-        # to remove the first value (typically zero).
-        refNumPoints = (
-            len(src.core.findAllAxialMeshPoints([refAssem], applySubMesh=True)) - 1
-        )
+
+        refNumPoints = len(src.core.findAllAxialMeshPoints([refAssem])) - 1
         allMeshes = []
         for a in src.core:
-            # Get the mesh points of the assembly, neglecting the first coordinate (typically zero).s
-            aMesh = src.core.findAllAxialMeshPoints([a], applySubMesh=True)[1:]
+            # Get the mesh points of the assembly, neglecting the first coordinate
+            # (typically zero).
+            aMesh = src.core.findAllAxialMeshPoints([a])[1:]
```

Comment:
Can you explain why we don't want the submeshes? I think DIF3D will still need these

Reply:
`applySubMesh=True` is the default. I was having trouble with the submeshes originally in downstream modules, but I believe it's resolved now. Trying to move forward with this current version, which includes the submesh.